src/databricks/labs/ucx/assessment/aws.py

```diff
@@ -282,6 +282,103 @@ def add_uc_role_policy(self, role_name, policy_name, s3_prefixes: set[str], acco
             return False
         return True
 
+    def add_migration_role(self, role_name: str) -> str | None:
```

Comment:
```suggestion
    def create_role(self, role_name: str) -> str | None:
```
`AWSResources` is 1-1 mapping with CLI, not our processes, do naming accordingly